Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:alpine
+
+RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app
+
+RUN mkdir -p /usr/local/lib/node_modules && chown -R node:node /usr/local/lib/node_modules
+
+WORKDIR /home/node/app
+
+COPY . .
+
+RUN npm install -g typescript
```

Comment:
Typescript should be installed locally with `npm install`. Is it really necessary to install it globally, too? I think all my dev machines have typescript installed globally, so I don't know if it is really needed. Could you confirm? If it is not needed, l. 5 would also have to be removed.

Reply:
You're right, thanks for the comments, I made the changes.